README.md

```diff
@@ -99,7 +100,11 @@ The following list represents the set of components (and props) that don't curre
 
 As new core components are released, which support referencing assets, we'll update this list to ensure users know what exactly they can expect to update using CodePush.
 
+*Note: CodePush works correctly with Video component only if using `require`. Example:*
 
+```javascript
+<Video source={require("foo")} />
```

Comment:
I'd also change the example here to actually include the file extension, ie `require("./foo.mp4")`

Reply:
Yes, it will be better!